protocol-spec/messages.md

```diff
@@ -69,17 +68,8 @@ Requests for new messages can be submitted to [the Buttplug Standard Github Issu
 
 ## Message Versioning
 
-In order to cope with changes to the schema across servers and clients that may not both support the same schema versions, each message has a message version number. These are based on the schema version they were introduced in, represented as an unsigned integer:
-
-* 0.1.0 =&gt; 0
-* 0.2.0 =&gt; 1
+In order to cope with changes to the schema across servers and clients that may not both support the same schema versions, each message type defined in this document has a message version number. These are based on the schema version they were introduced in, represented as an unsigned integer. As of version 1, this document and the schema are synchronized by using the message version over any other form of release number. Version 0 was represented as release 0.1.0.
 
 The message versions are not contained in the messages themselves, instead the client sends the overall schema message version as part of the RequestServerInfo message and the server includes its schema message version in the ServerInfo response. Note that the original version of RequestServerInfo does not have a parameter for this, so the 0.1.0 servers will reject clients capable of using the 0.2.0 schema. This is not seen as an issue, as the server is likely to be the first component in the Buttplug archtecture to be updated, and is the component most likely to be under the endd-user's control to update.
```

Comment:
"the 0.1.0 servers will reject clients capable of using the 0.2.0 schema"

This sentence doesn't match the new integer versioning.